e2e/playwright/copy-move-actions/src/tests/destination-picker-dialog.test.ts

```diff
@@ -0,0 +1,107 @@
+/*!
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { ApiClientFactory, MyLibrariesPage, NodesApi, SitesApi, test, Utils } from '@alfresco/playwright-shared';
+import { expect } from '@playwright/test';
+import { logger } from '@alfresco/adf-cli/scripts/logger';
```

Comment:
there should be other logger for tests, we should not import adf cli into the e2e.... alternatively can use just "console.error"

Reply:
Migrated logger from adf-cli to adf-testing. However, while looking into this, I also noticed that the same adf-cli logger is being used create-library.spec.ts as well as create-folder.spec.ts. If we want, I can update those imports as well. Just note that they are not at all related to the changes/objective of this PR